lxde-base/lxdm/lxdm-0.4.1-r10.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="2"
```

Comment:
Well, 0.4.1 is stable and I couldn't find any stable request for 0.5.3, therefore I'd state that someone running a system with stable keywords needs it. Both are broken with gtk-3.20, but that is another story.